.github/workflows/nightly-build.yml

```diff
@@ -2,7 +2,7 @@ name: Upload Nightly Binaries
 
 on:
   schedule:
-    - cron: '0 0 * * *' # run at mid night
```

Comment:
Will we going to delete the nightly build when its past X day (e.g 1 week)  - even if its out of scope for the PR

Reply:
It keeps overwriting the previous night, so I think this fine.